docs/en/functionalities.rst

```diff
@@ -0,0 +1,499 @@
+Functionalities
+################
+
+The IT-Wallet System provides Users with a simpler, faster, and more secure way to access services. This service is delivered through the use of a Wallet Solution, whose User Experience is structured into three main phases: pre-use, use, and post-use. 
+
+.. figure:: ../../images/UX-phases-usage.svg
+   :name: User Experience phases of Wallet usage
+   :alt: User Experience phases of Wallet usage
+   :width: 100%
+
+
+The following sections focus on the usage and post-usage phases. They define the functional requirements supporting the User Experience for the Activation, Acquisition, Presentation, Management, and Deactivation phases, along with interaction requirements related to error management, assistance requests, and feedback collection. 
+
+The Official Resources include recommendations on the required User-Wallet Instance interactions and design best practices that promote consistency across different Wallet Solutions in terms of how functionalities are accessed and used. 
+ 
+
+Activation of the Wallet Instance 
+**********************************
+
+Activation enables the User to access the Wallet Solution's functionalities for securely obtaining, presenting, and managing their Electronic Attestations. The activation process involves User Authentication with the Wallet Instance using their digital identity, which enables the generation of the PID. 
+
+Below are the functional requirements to support the User Experience related to the Activation of the Wallet Instance: 
+
+- the User MUST download the Wallet Solution onto their device to generate their Wallet Instance; 
+- the User MUST set up an unlocking method through a recognition mode (e.g., OTP, biometric authentication, Face ID, PIN). The User MUST use the selected recognition method each time they access the service to ensure security and protect their information; 
+- the User MUST review all relevant information regarding the activation process and service usage. Additionally, the User MUST read any policy from the Provider and PID Provider and/or the service's terms and conditions. The User MUST give their consent to proceed or MAY decline to cancel the operation; 
+- the User MUST select an Authentication option from those available; 
+- the User MUST complete the Authentication flow with the National Identity Provider’s service; 
+- the User MUST receive confirmation of the Authentication process outcome. If successful, the User MUST view a preview of their PID. The User MUST confirm the previewed information to proceed with Wallet Instance activation, or MAY cancel the operation; 
+- the User MUST authorize the operation using the unlocking method previously set; 
+- the User MUST receive confirmation of the successful activation of the Wallet Instance.
```

Comment:
Wallet Solution must provide to the user ... not the User, the USer is free unless he/she losts the wallet unit and then he/she MUST revoke it.